sbb-connector-webtest/src_test/com/axonivy/connector/sbb/webtest/trip/ITBookTrip.java

```diff
@@ -0,0 +1,316 @@
+
+    // Fill fields of InputTripSearchData dialog
+    $(By.id("form:from_input")).setValue("AA");
+    $(By.id("form:from_panel")).click();
+
+    $(By.id("form:to_input")).setValue("BB");
+    $(By.id("form:to_panel")).click();
+
+    $(By.id("form:time_input")).sendKeys(Keys.CONTROL, "A", Keys.DELETE);
+    $(By.id("form:time_input")).setValue("10:00");
+
+    // Proceed
+    $(By.id("form:proceed")).click();
+
+    // Assert
+    $(By.id("form:trips:0:departureDate")).shouldHave(text(DATE_FORMATTER_PRESENTABLE.format(currentDate)));
+    assertThat($(By.id("form:trips:0:departureTime")).innerHtml().matches("10:00  .")).isTrue();
+    assertThat($(By.id("form:trips:0:departureTime")).getAttribute("class").split(" ")).doesNotContain("red");
+    assertThat($(By.id("form:trips:0:arrivalDate")).exists()).isFalse();
+    $(By.id("form:trips:0:arrivalTime")).shouldHave(text("11:00"));
+    assertThat($(By.id("form:trips:0:arrivalTime")).getAttribute("class").split(" ")).doesNotContain("red");
+    $(By.id("form:trips:0:duration")).shouldHave(text("Duration: 01:00"));
+
+    assertThat($(By.id("icon")).getAttribute("class")).isEqualTo("fa-solid fa-train");
+
+    $(By.id("form:trips:0:segments:0:departureName")).shouldHave(text("Bern"));
+    $(By.id("form:trips:0:segments:0:departureDate")).shouldHave(text(DATE_FORMATTER_PRESENTABLE.format(currentDate)));
+    $(By.id("form:trips:0:segments:0:departureTime")).shouldHave(text("10:00"));
+    assertThat($(By.id("form:trips:0:segments:0:departureTime")).getAttribute("class").split(" "))
+            .doesNotContain("red");
+    $(By.id("form:trips:0:segments:0:departurePlatform")).shouldHave(text("Platform: 7"));
+    assertThat($(By.id("form:trips:0:segments:0:departurePlatform")).getAttribute("class").split(" "))
+            .doesNotContain("red");
+
+    assertThat($(By.id("form:trips:0:segments:0:arrivalName")).innerHtml().matches("Z.rich HB")).isTrue();
+    $(By.id("form:trips:0:segments:0:arrivalDate")).shouldBe(empty);
+    $(By.id("form:trips:0:segments:0:arrivalTime")).shouldHave(text("11:00"));
+    assertThat($(By.id("form:trips:0:segments:0:arrivalTime")).getAttribute("class").split(" "))
+            .doesNotContain("red");
+    $(By.id("form:trips:0:segments:0:arrivalPlatform")).shouldHave(text("Platform: 19"));
+    assertThat($(By.id("form:trips:0:segments:0:arrivalPlatform")).getAttribute("class").split(" "))
+            .contains("red");
+
+    $(By.id("form:trips:0:segments:0:transportProductName")).shouldHave(text("IC 721"));
+    $(By.id("form:trips:0:segments:0:direction")).shouldHave(text("Direction: St. Gallen"));
+
+    $(By.id("form:trips:0:onlineOffersLink")).shouldHave(text("Online Offers"));
+    assertThat($(By.id("form:trips:0:onlineOffersLink")).getAttribute("href")).isEqualTo(String.format(
+            """
+                    https://www.sbb.ch/en/buying/pages/fahrplan/fahrplan.xhtml?recon=wQg8CgJWMRKRBSVDMiVCNkhLSQkJgFQlMjRBJTNEMSU0ME8lM0RCZXJuJTQwWCU\

```

Comment:
hope this will be stable :)

Reply:
The sandbox URI which is used for testing seems to always return the same value for recon